### DISCUSSION TITLE
libGL error when running openroad -gui from within docker

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#3234] [General]
I'm running `openroad -gui` from within Docker and I get these errors. I think they are harmless, but the OpenROAD GUI sometimes locks up if I resize.

```
$ openroad -gui
libGL error: unable to load driver: swrast_dri.so
libGL error: failed to load driver: swrast
```

This is the error from OpenROAD GUI just as it locks up:

```
[WARNING GUI-0076] QXcbConnection: XCB error: 2 (BadValue), sequence: 1235, resource id: 1064, major code: 130 (Unknown), minor code: 3
```

Reproducing the lockup on the developer end has proven impossible so far, so this question is specifically about the error above.

**Comment by maliberty** [comment]
I've seen that warning before but it seems benign and has never cause any trouble.  I have no idea what it means but I doubt it relates to your missing libraries.
